index.ts

```diff
@@ -221,14 +219,18 @@ async function ensureWebhook(tableId: string) {
       },
     },
   };
-  const res = await api("POST",baseUrl, `bases/${state.BASE_ID}/webhooks`, null, JSON.stringify(body));
-  const { id } = await res.json();
+  const res = await api("POST", baseUrl, `bases/${state.BASE_ID}/webhooks`, null, JSON.stringify(body));
+  const webhook = await res.json();
 
-  if (id) {
-    await root.refreshWebhook({ id }).$invokeIn({ seconds: 60 * 60 * 24 * 6, key: id }); // refresh in 6 days
+  if (webhook.id) {
+    await root.refreshWebhook({ id: webhook.id }).$invokeIn({ seconds: 60, key: webhook.id }); // refresh in 6 days
```

Comment:
I'm guessing the `60` was for testing purposes

Reply:
exact